gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -59,7 +59,7 @@ int rotator_cc_impl::work(int noutput_items,
     const gr_complex* in = (const gr_complex*)input_items[0];
     gr_complex* out = (gr_complex*)output_items[0];
 
-#if 0
+#if 1
```

Comment:
What would be good here is if we could tell the VOLK version. Sadly, VOLK doesn't have a version.h file. We could get it from the GNU Radio cmake. Then, we could if based on if the VOLK version has the correct rotator.